python/sglang/srt/layers/moe/ep_moe/kernels.py

```diff
@@ -17,6 +17,116 @@
 logger = logging.getLogger(__name__)
 
 
+@triton.jit
+def compute_src2dst_triton_kernel(
```

Comment:
`compute_src2dst_triton_kernel` and `deepep_compute_src2dst_triton_kernel` are defined twice.

Reply:
It is equivalent to this pure PyTorch code:

```python
src2dst[reorder_ids] = torch.arange(num_toks, device=reorder_ids.device)
```

Why developing a triton kernel is necessary? Is it faster?